README.md

```diff
@@ -148,6 +148,15 @@ project.
 - [ ] Creating and managing Trust documents using [Trust Establishment](https://identity.foundation/trust-establishment/)
 - [ ] [DID Well Known Configuration](https://identity.foundation/.well-known/resources/did-configuration/) documents
```

Comment:
this can be a check now!

Reply:
lines 145,147, and 149 can all be checked

Reply:
@bobbilee19 able to update that ^